### DISCUSSION TITLE
Extra Themes

### DISCUSSION THREAD

**Discussion by IronCodem** [anuraghazra/github-readme-stats#943] [Q&A] [answered]
@anuraghazra, do you have to code the extra themes like `vue-dark` in order for them to appear?

**Comment by Itz-Hyperz** [comment, marked as answer]
In order to create a physical theme, you have to add the theme to the index.js file in the themes folder [here](https://github.com/anuraghazra/github-readme-stats/tree/master/themes). Then you need to make a pull request and wait for it to be merged.

If you just want to make your own theme, it's just adding more context to the URL of your readme stats. You can see more info on that [here](https://github.com/anuraghazra/github-readme-stats#customization).

```
If this helped you, I would appreciate it if you mark my comment as the answer :]
```
